src/amo/components/StaticPages/ReviewGuide/index.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+import Card from 'ui/components/Card';
+
+import 'amo/components/StaticPages/StaticPages.scss';
+
+
+export class ReviewGuideBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    /* eslint-disable react/no-danger */
+    return (
+
+      <Card
+        className="ReviewGuidePage"
+        header={i18n.gettext('Review Guidelines')}
+      >
+        <section>
+          <p>
+            {i18n.gettext(`Add-on reviews are a way for you to share your opinions about the add-ons you’ve installed and used. Our review moderation team reserves the right to refuse or remove any review that does not comply with these guidelines.`)}
+          </p>
+          <h2>{i18n.gettext('Some tips for writing a great review')}</h2>
+          <dl>
+            <dt className="staticSubHeading">{i18n.gettext('Do:')}</dt>
+            <dd>
+              <ul>
+                <li>{i18n.gettext('Write like you are telling a friend about your experience with the add-on.')}</li>
+                <li>{i18n.gettext('Keep reviews concise and easy to understand.')}</li>
+                <li>{i18n.gettext('Give specific and helpful details. For example:')}
+                  <ul>
+                    <li>{i18n.gettext('Did the add-on work as you expected it to?')}</li>
+                    <li>{i18n.gettext('What features did you like or dislike?')}</li>
+                    <li>{i18n.gettext('Was it useful?')}</li>
+                    <li>{i18n.gettext('Was it easy to use?')}</li>
+                    <li>{i18n.gettext('Will you continue to use this add-on?')}</li>
+                  </ul>
+                </li>
+                <li>{i18n.gettext('Take a moment to read your review before submitting it to minimize typos.')}</li>
+              </ul>
+            </dd>
+          </dl>
```

Comment:
You can remove the closing dl and opening dl and have a single definition list rather than one per question.